lib/companies/helper.ex

```diff
@@ -10,8 +10,9 @@ defmodule Companies.Helpers do
   end
 
   def sorted_list(list, params) do
-    {field, direction} = sorting_params(params)
-    Enum.sort_by(list, &Map.get(&1, field), direction)
+    params
+    |> sorting_params()
+    |> sort_by(list)
```

Comment:
The downside to using a list of values vs DB for the data source is that sorting is not as capable. Works for now but may need to rethink in the future.

Reply:
This is fun!